lib/src/local_working_copy.rs

```diff
@@ -1062,8 +1082,17 @@ impl TreeState {
         if clean {
             Ok(None)
         } else {
-            let new_file_type = new_file_state.file_type.clone();
             let current_tree_values = current_tree.path_value(repo_path);
+            #[cfg(windows)]
```

Comment:
nit: `if cfg!(windows)` as the code can be compiled on all platforms?

Reply:
Done.